app/src/js/actions/index.js

```diff
@@ -128,9 +128,9 @@ export const checkApiVersion = () => {
 };
 
 export const listCollections = (options = {}) => {
-  const { listAll, ...queryOptions } = options;
+  const { listAll = false, getMMT = true, ...queryOptions } = options;
   return (dispatch, getState) => {
-    const timeFilters = fetchCurrentTimeFilters(getState().datepicker);
+    const timeFilters = !listAll ? fetchCurrentTimeFilters(getState().datepicker) : {};
```

Comment:
nit: I would flip this to avoid the negation:
```javascript
const timeFilters = listAll ? {} : fetchCurrentTimeFilters(getState().datepicker);
```

Reply:
Yeah I like this better. Done.